source/server/http/admin.cc

```diff
@@ -758,7 +757,7 @@ void AdminImpl::createFilterChain(Http::FilterChainFactoryCallbacks& callbacks)
   callbacks.addStreamDecoderFilter(Http::StreamDecoderFilterSharedPtr{new AdminFilter(*this)});
 }
 
-Http::Code AdminImpl::runCallback(absl::string_view path_and_query,
+Http::Code AdminImpl::runCallback(absl::string_view path_and_query, AdminFilter* admin_filter,
```

Comment:
nit: `AdminFilter& admin_filter` (can't be null)

Reply:
done